publication-rest/build.gradle

```diff
@@ -58,5 +58,7 @@ test {
     environment "EXTERNAL_USER_POOL_URI", "https://external-user-pool.example.com"
     environment "API_HOST", "https://api-host.example.com"
     environment "CUSTOM_DOMAIN_BASE_PATH", "publication"
+    environment "IMPORT_CANDIDATES_TABLE", "import-candidates-table"
```

Comment:
Should table names be configurable from environment?
What is the use case?

Reply:
We read table name from environment to make ResourceService operate on specific table.

Reply:
How else would you pass in the table names?

Ex. "nva-import-candidates-master-pipelines-NvaPublicationApiPipeline-5W08RY2DNRIO-nva-publication-api" and  "nva-resources-master-pipelines-NvaPublicationApiPipeline-5W08RY2DNRIO-nva-publication-api" for 4 different aws accounts?

5W08RY2DNRIO is the stack name